src/fhirtypes/ElementDefinition.ts

```diff
@@ -1635,8 +1636,19 @@ export class ElementDefinition {
       'string',
       'uri'
     );
-    if (isEmpty(validTypes)) {
-      throw new CodedTypeNotFoundError(this.type ? this.type.map(t => t.code) : []);
+    const isBindable = this.type?.some(
+      type => fisher?.fishForMetadata(type.code, Type.Logical)?.canBind
+    );
+    if (isEmpty(validTypes) && !isBindable) {
+      if (this.type?.some(type => fisher?.fishForMetadata(type.code, Type.Logical) != null)) {
+        // Only warn if it was from a logical model that doesn't have #can-bind characteristic
+        logger.warn(
+          'Binding on a Logical Model type without the #can-bind Characteristic is discouraged',
```

Comment:
I think it's more than discouraged -- it's technically disallowed. Also, it looks like we usually do not capitalize _Logical Model_ in error messages and warnings. So maybe something more like:
```suggestion
          `Bindings can only be applied to logical model types with the #can-bind characteristic. Update the target logical model to declare the #can-bind characteristic or remove the binding from ${this.id}.`,
```